pkg/sdk/logging/model/output/kafka_test.go

```diff
@@ -69,6 +69,14 @@ brokers: kafka-headless.kafka.svc.cluster.local:29092
 default_topic: topic
 use_rdkafka: true
 ssl_verify_hostname: false
+rdkafka_options:
```

Comment:
This should also be changed, once struct changes are in place.

Also the expected config should also change, since new fields were added, shouldn't it? @pepov

Reply:
that is from librdkafka parameters(https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md). i think we should not do the changes.

Reply:
@csatib02 yes, the tests should break, we need to know why they didn't